internal/xcuit/config.go

```diff
@@ -0,0 +1,136 @@
+package xcuit
```

Comment:
Should we use the official framework name here too ?

Reply:
I renamed a package to `xcuitest`